netket/operator/_continuous_diffoperator.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+from netket.utils.types import DType
+
+from netket.hilbert import AbstractHilbert
+from netket.operator import AbstractOperator
+
+import jax
+import jax.numpy as jnp
+
+
+class DiffOperator(AbstractOperator):
```

Comment:
I would prefer a more explicit name, say DifferentialOperator (Diff is a bit ambiguous ?)

Reply:
Actually, where is this DiffOperator used?

Reply:
it is not used yet. but it is the more general interface which could replace the KineticPotential if the implementation is all right with you...

Reply:
I think we should sit down a moment and define the API of this interface on monday/tuesday.

Like, how it should behave under composition/addition/ecc so that we can define only one `expect/grad` rule that works for all continuous operators...

Reply:
I really like the idea, but I think that like this it is too crude?